test/fixtures/node_modules/module-t/package.json

```diff
@@ -0,0 +1,4 @@
+{
+    "main": "./main.js",
+    "browser": "module-b"
```

Comment:
yep

> On 10 Sep 2016, at 17:28, Roman Shtylman <notifications@github.com> wrote:
> 
> In test/fixtures/node_modules/module-t/package.json:
> 
> > @@ -0,0 +1,4 @@
> > +{
> > +    "main": "./main.js",
> > +    "browser": "module-b"
> this is the feature that previously did not work?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>